manifest.json

```diff
@@ -29,7 +29,7 @@
     
     "applications": {
         "gecko": {
-            "id": "kiwix-html5-unlisted@kiwix.org"
+            "id": "kiwix-js-unlisted@kiwix.org"
```

Comment:
At least for now, we need to keep this id untouched, else Firefox will probably consider the new versions as a different addon, and will probably won't do the upgrade for end-users